lib/httplib/httplib.go

```diff
@@ -173,8 +173,19 @@ func WithCSRFProtection(fn HandlerFunc) httprouter.Handle {
 }
 
 // ReadJSON reads HTTP json request and unmarshals it
-// into passed interface{} obj
-func ReadJSON(r *http.Request, val interface{}) error {
+// into passed any obj. A max of 10MB is allowed.
```

Comment:
Feels like the comment mentioning 10MB is prone to going out of date.

Should we just say that it enforces a reasonable maximum to prevent abuse?

Reply:
Done